common/perception_utils/include/perception_utils/perception_utils.hpp

```diff
@@ -102,6 +104,72 @@ inline double get2dIoU(const T1 source_object, const T2 target_object)
   return iou;
 }
 
+inline std::map<std::string, double> getMaxMin2d(const tier4_autoware_utils::Polygon2d & polygon)
```

Comment:
Generally, MinMax is better than MaxMin.

FYI. https://cpprefjp.github.io/reference/algorithm/minmax.html

Reply:
reflect the indication in https://github.com/autowarefoundation/autoware.universe/pull/1902/commits/975e98db152e6229c8f8f3cb674e93f98ff007ee